examples/layer-browser/src/app.js

```diff
@@ -102,6 +102,12 @@ class App extends PureComponent {
     setImmutableDataSamples(settings.immutable);
   }
 
+  _onQueryFeatures() {
```

Comment:
This isn't a handler, is it? I thought the intent here was to respond to an input event by calling `queryFeatures` (/elements/objects) manually from within the input event handler. We agreed this path would serve as a complement to the `onClick` / `onHover` callback-style interaction that we already have, to allow app devs more flexibility with how they handle input events.